app/js/view/details_view.js

```diff
@@ -35,14 +39,75 @@ export default class DetailsView extends View {
     });
   }
 
+  filterNonAffectedApps(apps, filters) {
+    return apps.filter(app => {
+      return filters.reduce((prev, cur) => {
+        return prev || cur.test(app.manifestURL);
```

Comment:
What this is doing is really obscured by the fact that it's being done behind a ```reduce()``` call. It's less code, but it's much clearer if we use a ```find()```.